services/plans/states/main.go

```diff
@@ -0,0 +1,10 @@
+package plan_states
```

Comment:
Maybe it would be better to put this file in a folder call "pass_plan_states" ? so that the file path is service/plans/pass_plan_states